content/docs/command-reference/params/diff.md

```diff
@@ -19,21 +19,24 @@ positional arguments:
 
 ## Description
 
-This command provides a quick way to compare parameter values among experiments
-in the repository history. Requires that Git is being used to version the
-project params.
+Provides a quick way to compare parameter values among experiments in the
+repository history. Requires that Git is being used to version the project
+params.
 
 > Parameter dependencies are defined with the `-p` option in `dvc run`. See also
 > `dvc params`.
 
 Without arguments, this command compares parameters currently present in the
 <abbr>workspace</abbr> (uncommitted changes) with the latest committed version.
+This includes everything in the default parameters file (`params.yaml`) as well
+as `params` found in `dvc.yaml` files.
```

Comment:
@shcheklein tagging you here as promised. Here's the clarification to your question in https://github.com/iterative/dvc.org/pull/2052#discussion_r549186721. I ended up reviewing all of the description here as well as in the index of params...

Reply:
`as `params` found in `dvc.yaml` files.` what does it exactly mean?

Reply:
"`params` found in `dvc.yaml` files" means the parameters tracked by DVC which may or may not match what's in the params file(s) in the workspace.

I added some missing context around this in ad461309.